### DISCUSSION TITLE
Microsoft MPI not read, WIndows 10

### DISCUSSION THREAD

**Discussion by aviony** [su2code/SU2#1270] [Q&A]
Hi,

I am trying to run SU2 on parallel, and MPI is installed and on path in system variables. I have also tried enforcing with meson with the commands said in the website, but the system does not take it either, as in it does not recognize meson commands also, when it is in system variables as well. 
When I installed SU2, it had shown the message of MPI not found. Is there any other way to make it work?

Thank you beforehand!

**Comment by pcarruscag** [comment]
Hi,
Apologies for the silence on our side, the truth is most of us do not run SU2 on Windows.
Even the binaries available with each release are created with cross compilation, not built on a Windows machine.
Usually (i.e. on Linux) when meson does not find something it is because pkg-config is not setup to look in the right directories, you can control this with the environment variable PKG_CONFIG_PATH.
If the serial compilation is working fine, and mpi is installed correctly, you can try using the windows equivalent of "export CC=mpicc" and "export CXX=mpicxx" before calling meson.py, and -Dcustom-mpi=true when calling it.

**Comment by aviony** [comment]
Hi Pedro,

Thank you for your reply. Apologies for my own late answer as well.

I am trying to install now the SU2 6.2.0 Falcon. It is the one I particularly need for a program that calls this program for Aeroelastic analyses. I turned from Windows to Linux Manjaro as this program apparently only works with MSL on Windows.
My C++ compiler is not doing the job when I use the command ./configure for the parallel build.  I have both cmake and gcc installed as compilers, but this is what happens:

[cid:ae2afe41-51cb-48da-a755-9fb054ea8537]

Which other C++ compilers can I use instead?

Thank you!

Best regards,

Avi
________________________________

**Comment by Casseydawn1994** [comment]
Okay so I'm new at all this and I'm just kinda figureing it out as I go on my own...  I've been at it for about 6 months and I've taught myself with the help of just reading how to do it now I'm in school but whenever I do any of those commands it doesn't work like the Dcustom and all those it's just not working 

Sent from Yahoo Mail on Android 
 
  On Tue, Apr 27, 2021 at 10:13 AM, ***@***.***> wrote:   

Hi Pedro,

Thank you for your reply. Apologies for my own late answer as well.

I am trying to install now the SU2 6.2.0 Falcon. It is the one I particularly need for a program that calls this program for Aeroelastic analyses. I turned from Windows to Linux Manjaro as this program apparently only works with MSL on Windows.
My C++ compiler is not doing the job when I use the command ./configure for the parallel build. I have both cmake and gcc installed as compilers, but this is what happens:

[cid:ae2afe41-51cb-48da-a755-9fb054ea8537]

Which other C++ compilers can I use instead?

Thank you!

Best regards,

Avi
________________________________

**Comment by danielenriquez59** [comment]
Wondering if anyone has had any luck with this?
Instead of mpicc, and mpicxx, Windows utilizes mpiexec. 
My mpiexec is installed at "Program Files\Microsoft MPI\Bin"

Should I set my PKG_CONFIG_PATH env variable to this directory? And then how would I tell meson to build with mpiexec instead of linux equivalents.
